src/Twig/CommitteeRuntime.php

```diff
@@ -92,10 +93,36 @@ public function isVotingCommittee(Adherent $adherent, Committee $committee): boo
         return $membership && $membership->isVotingCommittee();
     }
 
+    public function getVotingCommittee(CommitteeMembershipCollection $memberships)
+    {
+        $votingCommittee = '';
+
+        foreach ($memberships as $membership) {
+            if ($membership->isVotingCommittee()) {
+                $votingCommittee = $membership->getCommittee()->getSlug();
```

Comment:
to be consistent with the method name, could we return `Committee` instance?
or rename in `getVotingCommitteeName` and return the name?